theme/static/js/hs-vue/subject-keywords-app.js

```diff
@@ -6,13 +6,17 @@ let subjKeywordsApp = new Vue({
     components: {
         VueBootstrapTypeahead
     },
-    delimiters: ['${', '}'],
     data: {
```

Comment:
this pattern never should have been introduced, it's confusing to have in massive templates that get combined with jQuery code that also uses $ simple alternate Django template language has been engaged